packages/wrangler/src/whoami.ts

```diff
@@ -52,6 +61,40 @@ export async function whoami() {
 	}
 }
 
+async function printMembershipInfo(user: UserInfo, accountFilter?: string) {
+	try {
+		if (!accountFilter) {
+			return;
+		}
+		const eq = (a: string, b: string) => a.localeCompare(b, undefined, { sensitivity: "base" }) == 0; // prettier-ignore
+		const selectedAccount = user.accounts.find(
+			(a) => eq(a.id, accountFilter) || eq(a.name, accountFilter)
+		);
+		if (!selectedAccount) {
+			return;
+		}
+		const membershipRoles = await fetchMembershipRoles(selectedAccount.id);
+		if (!membershipRoles) {
+			return;
+		}
+		logger.log(
+			`🎢 Membership roles in "${selectedAccount.name}": Contact account admin if you need more roles.`
```

Comment:
Copy suggestion: 🎢 Membership roles in "${selectedAccount.name}": Contact account admin if you lack permissions for your work.

Reply:
Membership roles in "${selectedAccount.name}": Contact your account's super admin to change your permissions.